ignite/cmd/plugin_default.go

```diff
@@ -0,0 +1,95 @@
+package ignitecmd
```

Comment:
It should be definitely simple to have `network` plugin imported.

Didn't we want to have a special prompt to import easily any plugins in Ignite awesome?

Reply:
We want the transition for users to be as seamless as possible for `network`.  

If they use `v0.26.0` and call `ignite network` they will just be prompted and then have normal access to the commands like they expected.

Reply:
> We want the transition for users to be as seamless as possible for network.

Yes it's sophisticated because of that.

If we don't do that, we'll have users complaining "ouin `network` command is no longer available!" and you will have to answer "please run `ignite install default-plugins`" each time... We could eventually output that on `ignite network` if the plugin isn't installed, but that will not remove a lot of complexity here.

Reply:
@aljo242 Just to add, I don't remember if we stated this, but users aren't prompted on `ignite network`, the plugin is downloaded automatically.